src/cursor/cur_file.c

```diff
@@ -1203,18 +1203,15 @@ __wt_curfile_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *owner, c
     if (!F_ISSET(S2C(session), WT_CONN_IN_MEMORY)) {
         WT_RET(__wt_config_gets_def(session, cfg, "bulk", 0, &cval));
         if (cval.type == WT_CONFIG_ITEM_BOOL ||
-          (cval.type == WT_CONFIG_ITEM_NUM && (cval.val == 0 || cval.val == 1))) {
-            bitmap = false;
+          (cval.type == WT_CONFIG_ITEM_NUM && (cval.val == 0 || cval.val == 1)))
```

Comment:
Was the `bitmap` config removed in a previous PR? If not, should it be banished in this one?

Reply:
Sure, but if there's no reason to keep it, it should go. I won't let it block the PR but I would like to see it gone.